odoo/addons/base/ir/ir_actions.py

```diff
@@ -1175,9 +1176,11 @@ class IrActionsActClient(models.Model):
     def _compute_params(self):
         self_bin = self.with_context(bin_size=False, bin_size_params_store=False)
         for record, record_bin in zip(self, self_bin):
-            record.params = record_bin.params_store and safe_eval(record_bin.params_store, {'uid': self._uid})
+            params = record_bin.params_store and base64.b64decode(record_bin.params_store)
+            record.params = params and safe_eval(params, {'uid': self._uid})
```

Comment:
Wait a second. If I am not mistaken, this will crash on an existing database,
since the existing values are *not* base64-encoded!
This goes against the rules of stable policy on major versions.

Reply:
@nim-odoo can you comment on this?
Thanks!

Reply:
@rco-odoo, Right it will definitely affect the existing database. let me check if we have a better solution for stable. Thanks